src/startupscriptgenerator/src/common/buildManifestHelper.go

```diff
@@ -7,49 +7,76 @@ package common
 
 import (
 	"common/consts"
-	"log"
+	"fmt"
+	"os"
 	"path/filepath"
 
 	"github.com/BurntSushi/toml"
 )
 
 type BuildManifest struct {
```

Comment:
Some members in this struct end with `File`, some end with `FileName` - can we align the names to the same convention?

Reply:
I agree regarding the naming differences, but I do not want to handle that in this PR because it requires a change in the build phase too and we need to consider backcompat scenarios too.